docs/modules/ROOT/pages/documentation/extensions/autoscale/README.adoc

```diff
@@ -0,0 +1,110 @@
+
+#### The Base Admin Console Plugin
+This module contains common handler methods, the tab group that admin console plugin extensions should attach to, and
+a generic "Active AutoScale Group" page from which a user can see which AutoScale group is attached to a specific
+Deployment Group and scale it up or down.
+
+### Extension Modules
+
+Extensions are intended to be composed of 2 parts: an implementation module, and an admin console plugin.
+
+### The Implementation Module
+The implementation module is where the admin commands, services, and config bean interfaces should exist. The following
+is expected as a minimum:
+
+* A ConfigBean extension for `ScalingGroup` - This ConfigBean is the "type" of AutoScale Group, and as such should
+contain any extension specific attributes and elements
+* A service extension for `Scaler` - This service should implement the `scaleUp` and `scaleDown` methods, and should be
+qualified with the `Scales` annotation pointing at your `ScalingGroup` ConfigBean.
+* Create command - An admin command used to create a scaling group of your extension's type.
+* Get command - An admin command used to get the configuration of a scaling group of your extension's type. This command
+is expected to be in the format of `get-${ScalingGroupExtensionName}-configuration`
+(e.g. `get-nodes-scaling-group-configuration`)
+* Set command - An admin command used to set the configuration of a scaling group of your extension's type. This command
+is expected to be in the format of `set-${ScalingGroupExtensionName}-configuration`
+(e.g. `set-nodes-scaling-group-configuration`)
+
+### The Admin Console Plugin
+
+The admin console plugin is where you add your tabs and pages for listing, creating, deleting, and editing AutoScale
+groups of your extension's type. The "base" admin console plugin provides a tab group integration point that you can add
+your extension tabs to named `fish.payara.admingui.extensions:autoScaleGroupsTab` (done via your extension's
+`console-config.xml` file).
+
+== Installing AutoScale Extensions
+
+Regardless of which AutoScale extension you wish to install, it is expected that the API and the two Core module JARs
+are added to the server, by dropping them into the `${GLASSFISH_HOME}/modules` directory of your Payara Server
+installation.
+
+You can then drop the implementation and admin console plugin JARs of your desired extensions to the same directory.
+
+For example, to install the SSH Nodes AutoScale Extension, you would drop the following JARs into the `modules`
+directory:
+
+* _autoscale-groups-api_
+* _autoscale-groups-core_
+* _autoscale-groups-console_
+* _autoscale-groups-nodes-plugin-core_
+* _autoscale-groups-nodes-plugin-console_
+
+You must restart the server if it was running to see your extension activated.
+
+TIP: Make sure that, if the AutoScale extension has dependencies in order to run, the dependency artifacts are also in
```

Comment:
```suggestion
IMPORTANT: Make sure that, if the AutoScale extension has dependencies in order to run, the dependency artifacts are also in
```

A "tip" to me is something that is an optional trick we suggest to use but isn't necessary. This sounds like a necessary thing otherwise the extension wouldn't work.